datalad/local/add_readme.py

```diff
@@ -236,5 +241,8 @@ def __call__(filename='README.md',
                 fpath,
                 message='[DATALAD] added README',
                 result_filter=None,
-                result_xfm=None):
+                result_xfm=None,
+                return_type='generator',
+                result_renderer='disabled'
+        ):
```

Comment:
The whole call could be reduced to `yield from` instead of the `for` loop

Reply:
thanks, done!